CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.17.0 (Unreleased)
 - [Enhancement] Update `librdkafka` to `2.4.0`
 - [Feature] Add `#seek_by` to be able to seek for a message by topic, partition and offset (zinahia)
+- [Change] Remove old producer timeout API warnings.
```

Comment:
@mensfeld This is a breaking change but not marked as a breaking change :( 

This MR removes warnings as well as changes the API

Reply:
Yeah, you are right. Sorry :man_shrugging: 

This lib is still pre 1.0 so technically the APIs are not stable but I do admit this should be communicated better :pray: